### PR TITLE
Update Store & User Models

### PR DESCRIPTION
**Pull request by revelted**

### CODE REVIEW

File: server/db/models/store.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    },
+    address: {
+      street: String,
+      city: String,
+      state: {
+        type: String,
+        uppercase: true,
+        required: [true, "Must write similar to 'FL'"],
+        maxlength: 2
+      },
```
Comment (leopolicastro) [inline server/db/models/store.js:18]: street city and state should be its own attribute

File: server/db/models/store.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    },
+    serviceMenu: {
+      type: mongoose.Schema.Type.ObjectId,
+      ref: 'Dish'
+    },
+    themedMealSet: {
+      type: mongoose.Schema.Type.ObjectId,
+      ref: 'MealSet'
+    },
+    availabilityCalender: {
```
Comment (leopolicastro) [inline server/db/models/store.js:52]: typo here

File: server/db/models/store.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    }
+  },
+  { timestamps: true }
+);
+
+storeSchema.methods.toJSON = function () {
+  const store = this;
+  const storeObject = store.toObject();
+  return storeObject;
+};
```
Comment (leopolicastro) [inline server/db/models/store.js:101]: this doesn't seem to be doing anything.

**Review by leopolicastro — CHANGES_REQUESTED** [review]

File: server/db/models/store.js (hunk 1 above)
Comment (JespsonSTP) [reply server/db/models/store.js:18]: this is so it can be easier when using geolocation api. we can get all that information in attribute

**Pull request merged**